### ISSUE TITLE
Flags without references get deleted when you enter play mode

### ISSUE THREAD

**Issue by daquinte** [e-ucm/uAdventure#40]
When you create a flag and you have zero references, and you enter play mode the flag will get destroyed. As a side note, the whole interface updates when you enter and exit playmode, which is something that can be confusing.

**Comment by Victorma** [comment]
Yes, this happens because flags are not stored anywhere. The flags and variables are stored where they are used. This means that if a flag is not used, it wont exist the next time the project reloads. 

However, we might be able to store them in the project descriptor in case they're not refferenced to be saved between reloads.

The playmode editor window refresh happens as the project gets reloaded when the project is played. 

Both features will be impemented in future versions, but not for the v0.12.
